runtime/interpreter/value_publickey.go

```diff
@@ -89,32 +71,5 @@ func NewPublicKeyValue(
 		})
 	}
 
-	// Public key value to string should include the key even though it is a computed field
-	publicKeyValue.Stringer = func(
-		memoryGauge common.MemoryGauge,
-		publicKeyValue *CompositeValue,
-		seenReferences SeenReferences,
-	) string {
-
-		stringerFields := []CompositeField{
-			{
-				Name:  sema.PublicKeyTypePublicKeyFieldName,
-				Value: publicKey,
-			},
-			{
-				Name: sema.PublicKeyTypeSignAlgoFieldName,
-				// TODO: provide proper location range
-				Value: publicKeyValue.GetField(interpreter, EmptyLocationRange, sema.PublicKeyTypeSignAlgoFieldName),
-			},
-		}
-
-		return formatComposite(
-			memoryGauge,
-			string(publicKeyValue.TypeID()),
-			stringerFields,
-			seenReferences,
-		)
-	}
-
```

Comment:
The custom stringer is actually not needed, the default string formatting function produces the same output now